code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -583,14 +583,36 @@
 		/obj/item/clothing/glasses/hud/health = 4,
 		/obj/item/storage/firstaid/regular = 4,
 		/obj/item/storage/firstaid/adv = 4,
-		/obj/item/storage/pouch/medical = 4,
-		/obj/item/storage/pouch/medkit = 4,
-		/obj/item/storage/pouch/magazine/large = 4,
-		/obj/item/storage/pouch/magazine/pistol/large = 4,
+		/obj/item/storage/pouch/medical = 8,
+		/obj/item/storage/pouch/medkit = 8,
+		/obj/item/storage/pouch/magazine/large = 8,
+		/obj/item/storage/pouch/magazine/pistol/large = 8,
+		/obj/item/storage/pouch/pistol = 8,
+		/obj/item/storage/syringe_case = 16,
 		/obj/item/clothing/mask/gas = 4,
-		/obj/item/storage/pouch/pistol = 4,
+		/obj/item/storage/syringe_case/combat = 4,  //these have the regular combat injectors so dont change it to adv without paying attention
+		/obj/item/storage/pill_bottle/bicaridine = 4,
+		/obj/item/storage/pill_bottle/kelotane = 4,
+		/obj/item/storage/pill_bottle/tramadol = 4,
+		/obj/item/storage/pill_bottle/tricordrazine = 4,
+		/obj/item/storage/pill_bottle/alkysine = 4,
+		/obj/item/storage/pill_bottle/imidazoline = 4,
+		/obj/item/storage/pill_bottle/iron = 4,   //we will add more if needed later
+		/obj/item/stack/medical/advanced/bruise_pack = 16,
+		/obj/item/stack/medical/advanced/ointment = 16,
+		/obj/item/stack/medical/splint = 8,
+		/obj/item/reagent_containers/glass/beaker = 8, //so medics can make SMALL mixes in their quarters
+		/obj/item/reagent_containers/glass/bottle/bicaridine = 4,
+		/obj/item/reagent_containers/glass/bottle/kelotane = 4,
+		/obj/item/reagent_containers/glass/bottle/tramadol = 4,
+		/obj/item/reagent_containers/glass/bottle/inaprovaline = 4,
+		/obj/item/reagent_containers/glass/bottle/dylovene = 4,
+		/obj/item/reagent_containers/hypospray/autoinjector/dexalinplus = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/quickclotplus = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/synaptizine = 0,
```

Comment:
Why are you adding these in the first place then

Reply:
to allow restocking of said med

Reply:
Remove it